src/ImageProcessor.Web/HttpModules/ImageProcessingModule.cs

```diff
@@ -214,12 +214,32 @@ public static void SetHeaders(HttpContext context, string responseType, string[]
 
             cache.SetExpires(DateTime.Now.ToUniversalTime().AddDays(maxDays));
             cache.SetMaxAge(new TimeSpan(maxDays, 0, 0, 0));
-            cache.SetRevalidation(HttpCacheRevalidation.AllCaches);
+            //cache.SetRevalidation(HttpCacheRevalidation.AllCaches);
```

Comment:
I added it back in. It shouldn't be commented out as it prevents the browser checking the cache for changes.